packages/workbox-core/internal/models/messages/messageGenerator.mjs

```diff
@@ -12,14 +12,14 @@ const fallback = (code, ...args) => {
 const generatorFunction = (code, ...args) => {
   const message = messages[code];
   if (!message) {
-    core.INTERNAL.logHelper.warn(`Unable to find message for code '${code}'`);
+    logger.warn(`Unable to find message for code '${code}'`);
```

Comment:
As a user, if logger was unable to find a message for a specific code, what's the next actionable step?

Reply:
It would basically be a case of raising an issue with this and hopefully we can figure out which use of the code is invalid and help from there.

We can (and should) probably print the args as well, will PR that in.

Do think it would be worth adding a message asking developers to raise an issue in this case?